splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -2544,7 +2544,6 @@
     "AnyValidator":{
       "type": "array",
       "description": "It is used to validate the values of fields using various validators.",
-      "minItems": 1,
```

Comment:
Can you also adjust types for ui?

just remove this one line [here](https://github.com/splunk/addonfactory-ucc-generator/blob/develop/ui/src/types/globalConfig/entities.ts#L101).

Reply:
Thanks. Done